### PR TITLE
New website design

### PR DESCRIPTION
**Pull request by muuvmuuv**
A new pull request to merge some commits.

Related to: #122

### CODE REVIEW

File: website/static/img/languages/html.svg [hunk 1]
```diff
@@ -1,8 +1 @@
...
-	<path fill="#e44d26" d="M107.6 471l-33-370.4h362.8l-33 370.2L255.7 512"/>
-	<path fill="#f16529" d="M256 480.5V131H404.3L376 447"/>
-	<path fill="#ebebeb" d="M142 176.3h114v45.4h-64.2l4.2 46.5h60v45.3H154.4M156.4 336.3H202l3.2 36.3 50.8 13.6v47.4l-93.2-26"/>
-	<path fill="#fff" d="M369.6 176.3H255.8v45.4h109.6M361.3 268.2H255.8v45.4h56l-5.3 59-50.7 13.6v47.2l93-25.8"/>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><title>HTML5 Logo</title><path d="M108.4 0h23v22.8h21.2V0h23v69h-23V46h-21v23h-23.2M206 23h-20.3V0h63.7v23H229v46h-23m53.5-69h24.1l14.8 24.3L313.2 0h24.1v69h-23V34.8l-16.1 24.8-16.1-24.8V69h-22.6m89.2-69h23v46.2h32.6V69h-55.6"/><path fill="#e44d26" d="M107.6 471l-33-370.4h362.8l-33 370.2L255.7 512"/><path fill="#f16529" d="M256 480.5V131h148.3L376 447"/><path fill="#ebebeb" d="M142 176.3h114v45.4h-64.2l4.2 46.5h60v45.3H154.4m2 22.8H202l3.2 36.3 50.8 13.6v47.4l-93.2-26"/><path fill="#fff" d="M369.6 176.3H255.8v45.4h109.6m-4.1 46.5H255.8v45.4h56l-5.3 59-50.7 13.6v47.2l93-25.8"/></svg>
```
Comment (Glavin001) [inline website/static/img/languages/html.svg]: This is a formatting change to **undo** the previous formatting? 🤔

**Comment by muuvmuuv** [comment]
Ah okay. Now I understand what @Glavin001 meant. Okay, those changes are mostly necessary. And all those css are from me and just so many because I wanted to split elements in files for better maintaining. As this is a complete redesign I dont think I can split this in more PR because the most parts depent on each other. 
And I did not removed formatting for svg images, maybe by acident. I‘ll put it back after my vacations (two weeks). I think that is a lot of time to review most parts. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Glavin001** [comment]
> And all those css are from me and just so many because I wanted to split elements in files for better maintaining. 

OK I do see now these files are relatively small and are mostly -- if not all -- CSS. I trust your CSS abilities, likely more than my own 😆 , so I'm OK approving the bulk of these CSS changes 👍 .

> And I did not removed formatting for svg images, maybe by acident.

Likely by accident. I want to make sure you did not make any changes though. Can we revert all of those SVG re-formatting changes or should you sift through them to check for real changes first?

> I‘ll put it back after my vacations (two weeks). I think that is a lot of time to review most parts.

Are you leaving for vacation now? I hope you have a great time! 🎉 I'll try to review this over the upcoming week. May even be merged by the time you're home 😃 . Great work and thanks again for contributing! Enjoy your well deserved vacation!!!

**Comment by muuvmuuv** [comment]
What do you mean by changes of the svgs? The most of the are optimized manually.
Yep on vacation since yesterday. Happy coding :) 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Glavin001** [comment]
> What do you mean by changes of the svgs?

The SVGs were un-formatted.

**Comment by Glavin001** [comment]
Awesome work @muuvmuuv ! I've merged this in, with a few fixes to /users and /help pages. Please open more PRs if you have changes you'd like to make. Thanks again for contributing!! 🎉

**Pull request merged**